src/style.scss

```diff
@@ -14,7 +14,7 @@
   display: none;
 }
 
-auro-button {
+[part="button"] {
```

Comment:
Why did you change this to `part="button"` instead of just doing `[auro-button]` like you did to the icon?

Reply:
Because the latest button isn't doing the attribute addition for custom names.